frame/examples/offchain-worker-ping-pong/README.md

```diff
@@ -0,0 +1,53 @@
+<!-- markdown-link-check-disable -->
```

Comment:
Did you write this, or is this file being auto-generated?

Reply:
it came from `master`: https://raw.githubusercontent.com/paritytech/substrate/master/frame/examples/offchain-worker/README.md